bundles/framework/layerselector2/view/LayersTab.js

```diff
@@ -356,8 +360,11 @@ Oskari.clazz.define(
                 if (group.layerListPanel.isVisible()) {
                     visibleGroupCount += 1;
                 }
-                group.layerListPanel.setTitle(group.getTitle() + ' (' +
-                    visibleLayerCount + '/' + layers.length + ')');
+                group.layerListPanel.setTitle(group.getTitle());
```

Comment:
Title doesn't need to be updated anymore?

Reply:
It only needs to know the group title, the filtered layers / layers is shown in the badge.

Reply:
Sure, I meant that you can probably drop the whole setTitle() call from here.